drivers/i2c/i2c_nrfx_twi.c

```diff
@@ -12,6 +12,8 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(i2c_nrfx_twi, CONFIG_I2C_LOG_LEVEL);
 
+#define NRFX_I2C_TIMEOUT_MSEC		K_MSEC(100)
```

Comment:
This macro should not start with `NRFX_` as this prefix should be reserved for things that come from nrfx.

Reply:
Changed to `I2C_TRANSFER_TIMEOUT_MSEC`